src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/integration/visits/VisitsMigrationIntTest.kt

```diff
@@ -85,28 +90,34 @@ class VisitsMigrationIntTest : IntegrationTestBase() {
           assertThat(it.toDateTime).isEqualTo(LocalDateTime.parse("2020-01-02T23:30:00"))
         }
       )
-      verify(visitsMigrationService).migrateVisitsByPage(
+      verify(visitsMigrationService).divideVisitsByPage(
         check {
           assertThat(it.body.prisonIds).containsExactly("MDI", "BXI")
           assertThat(it.body.visitTypes).containsExactly("SCON", "OFFI")
           assertThat(it.body.fromDateTime).isEqualTo(LocalDateTime.parse("2020-01-01T01:30:00"))
           assertThat(it.body.toDateTime).isEqualTo(LocalDateTime.parse("2020-01-02T23:30:00"))
-          assertThat(LocalDateTime.parse(it.migrationId)).isCloseTo(LocalDateTime.now(), within(10, ChronoUnit.SECONDS))
-          assertThat(it.estimatedCount).isEqualTo(23_045)
+          assertThat(LocalDateTime.parse(it.migrationId)).isNotNull
```

Comment:
decided to just check we have a migrationId rather then checking the actual value which happens to be a date which was beginning to be a bit flaky given how long the test takes to run